### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by Kennywiseisdead** [InZidiuZ/op-framework-issue-tracker#1778]
**Assurance:**
Have you read through the rules from the `README.md` file in the root folder of this GitHub repository?

**Player Identification:**
What is your steam profile?

**Summary:**
Write a short summary of what you are experiencing.

**Steps to Reproduce:**
Include what steps that are needed in order to reproduce this issue.

**Expected Results:**
What are the expected results?

**Actual Results:**
What are the actual results?

## Additional Information

**Screenshots:**
If suitable, include some screenshots.

**Game Resolution:**
What is your game's resolution?

**Player Data:**
Write '/info' whilst in-game and provide a screenshot of the information you are prompted.

**Server:**
What server/community did this issue occur in?

**Notes:**
For anything else noteworthy.

**Comment by Kennywiseisdead** [comment]
I found two bugs with two vehicles. 1. Elegy Retro custom. 2. Benefector Stinling GT

1. In my elegy R custom, I installed neon lights a long time ago. But from 2 days it is not glowing. I had to go to the custom shop and install it again. But again after storing and taking the vehicle out from garage it is not glowing.
2. In my Stinling GT sports classic, The vehicle is not moving, the tyres are just burning out, but not moving. Feels like the front brake is applied. Not able to drive it
